lib-persistence-manager/persistencemgr/redis.go

```diff
@@ -394,7 +389,7 @@ func (p *ConnPool) Update(table, key string, data interface{}) (string, *errors.
 	}
 	writeConn := writePool.Get()
 	defer writeConn.Close()
-	_, createErr := writeConn.Do("SET", saveID, jsondata)
+	_, createErr := writeConn.Do("SETNX", saveID, jsondata)
```

Comment:
should this be SETXX or SETNX, check once

Reply:
we should use XX for this